src/plugins/dashboard/public/application/dashboard_state.test.ts

```diff
@@ -48,6 +52,23 @@ describe('DashboardState', function () {
     });
   }
 
+  function initDashboardContainer(initialInput?: Optional<DashboardContainerInput>) {
```

Comment:
In this case `Partial` isn't the same thing as `Optional`?

Reply:
Good catch - all the props should be optional so I've switched this to `Partial` for simplicity.